src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -200,19 +201,59 @@
+           on-press-to-network]}]
+
+  (let [token-symbol                                   (:symbol token)
+        loading-suggested-routes?                      (rf/sub
+                                                        [:wallet/wallet-send-loading-suggested-routes?])
+        from-values-by-chain                           (rf/sub
+                                                        [:wallet/wallet-send-from-values-by-chain])
+        to-values-by-chain                             (rf/sub [:wallet/wallet-send-to-values-by-chain])
+        suggested-routes                               (rf/sub [:wallet/wallet-send-suggested-routes])
+        selected-networks                              (rf/sub [:wallet/wallet-send-selected-networks])
+        disabled-from-chain-ids                        (rf/sub
+                                                        [:wallet/wallet-send-disabled-from-chain-ids])
+        routes                                         (when suggested-routes
+                                                         (or (:best suggested-routes) []))
+        {token-balances-per-chain :balances-per-chain} (rf/sub
+                                                        [:wallet/current-viewing-account-tokens-filtered
+                                                         (str token-symbol)])
+        affordable-networks                            (send-utils/find-affordable-networks
+                                                        {:balances-per-chain token-balances-per-chain
+                                                         :input-value        input-value
+                                                         :selected-networks  selected-networks
+                                                         :disabled-chain-ids disabled-from-chain-ids})
+        network-links                                  (if loading-suggested-routes?
+                                                         affordable-networks
+                                                         routes)
+        show-routes?                                   (or (and (not-empty affordable-networks)
+                                                                loading-suggested-routes?)
+                                                           (not-empty routes))]
+
+    (rn/use-effect
+     #(when (> (count affordable-networks) 0)
+        (fetch-routes input-value routes-can-be-fetched? 2000))
+     [input-value routes-can-be-fetched?])
+    (rn/use-effect
+     #(when (> (count affordable-networks) 0)
+        (fetch-routes input-value routes-can-be-fetched? 0))
+     [disabled-from-chain-ids])
+    (if show-routes?
       (let [initial-network-links-count   (count network-links)
-            disabled-count                (count disabled-from-networks)
-            network-links                 (if (not-empty disabled-from-networks)
+            disabled-count                (count disabled-from-chain-ids)
```

Comment:
This change is right here?
Cc @briansztamfater

Reply:
I think the idea of moving subscriptions to this file is okay. However, this component will be simplified in #19696. The current implementation is pretty messy, so I wouldn't be too picky about reviewing this file for now. We should just be careful not to break the existing functionality.

Reply:
Cool, sounds good to me👌

Reply:
@J-Son89, should be correct because before refactoring `disabled-from-networks` was a property in top component passed to `routes` component and it was initialized as `:disabled-from-networks disabled-from-chain-ids`